src/test/java/seedu/guest/logic/parser/EditCommandParserTest.java

```diff
@@ -108,12 +115,12 @@ public void parse_invalidValue_failure() {
     @Test
```

Comment:
For the line 111 above should we add VALID_DATE_RANGE_AMY as well for consistency?

Reply:
Yes, fixed